js/storage.js

```diff
@@ -9,22 +9,22 @@ function getStorageTasks() {
     }
     catch {
         jsonTasks = [
-            new Task("Homework", "Do my homework", false, "h", "default", new Date(2021, 6, 21)),
-            new Task("Breakfast", "Have a nice breakfast", false, "m", "default", new Date(2021, 7, 21)),
-            new Task("Music", "Listen to music", false, "l", "default", new Date(2021, 8, 21))
+            new Task("Homework", "Do my homework", false, "h", new Date(2021, 6, 21)),
+            new Task("Breakfast", "Have a nice breakfast", false, "m", new Date(2021, 7, 21)),
+            new Task("Music", "Listen to music", false, "l", new Date(2021, 8, 21))
         ]
     }
     finally {
         if (!jsonTasks) {
             jsonTasks = [
-                new Task("Homework", "Do my homework", false, "h", "blue", new Date(2021, 6, 21)),
-                new Task("Breakfast", "Have a nice breakfast", false, "m", "default", new Date(2021, 7, 21)),
-                new Task("Music", "Listen to music", false, "l", "default", new Date(2021, 8, 21))
+                new Task("Homework", "Do my homework", false, "h", new Date(2021, 6, 21)),
+                new Task("Breakfast", "Have a nice breakfast", false, "m", new Date(2021, 7, 21)),
+                new Task("Music", "Listen to music", false, "l", new Date(2021, 8, 21))
```

Comment:
create `generateFakeData` helper or static method in `Store` class for this (name should not be the same 😄 )

Reply:
Thanks, I created a class `Storage` and method `generateFakeData()`. I like your naming 😅
```suggestion
    export default class Storage {
        generateFakeData() {
            const fakeTasks = [
                new Task("Homework", "Do my homework", false, "h", new Date(2021, 6, 21)),
                new Task("Breakfast", "Have a nice breakfast", false, "m", new Date(2021, 7, 21)),
                new Task("Music", "Listen to music", false, "l", new Date(2021, 8, 21))
            ];
    
            this.updateStorageTasks(fakeTasks);
            return fakeTasks;
        }
        ...
```